app/Http/Controllers/Post/PostController.php

```diff
@@ -150,6 +150,38 @@ public function show($id)
     public function indexAdmin()
     {
         $posts = $this->postRepository->paginate(config('paginate.admin.post'));
-        return view('admin.post.publish', compact('posts'));
+        return view('admin.post.list', compact('posts'));
+    }
+
+    /**
+     * Publish post.
+     *
+     * @param \Illuminate\Http\Request $request description
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function publish(Request $request)
+    {
+        if ($request->status == true) {
+            $this->postRepository->unpublishPost($request->id);
+        } else {
+            $this->postRepository->publishPost($request->id);
+        }
+        Session::flash('message', trans('post.message.status'));
+        return back();
+    }
+
+    /**
+     * Remove the specified resource from storage.
+     *
+     * @param int $id id permission
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id)
+    {
+        $this->postRepository->delete($id);
```

Comment:
post_tab will auto sync. About comments, rattings, like_posts i will delete them when i delete post. Because i have not created table for them, i will update method destroy() in PostController when i created that tables.